test/extended/etcd/vertical_scaling.go

```diff
@@ -54,6 +55,64 @@ var _ = g.Describe("[sig-etcd][Feature:EtcdVerticalScaling][Suite:openshift/etcd
 		err = errors.Wrap(err, "pre-test: timed out waiting for initial cluster state to have 3 running machines and 3 voting members")
 		o.Expect(err).ToNot(o.HaveOccurred())
 
+		cpmsSupported, err := scalingtestinglibrary.PlatformSupportsCPMS(ctx, g.GinkgoT(), cpmsClient)
+		err = errors.Wrap(err, "pre-test: failed to determine if ControlPlaneMachineSet is present")
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		if cpmsSupported {
+
+			// TODO: Add cleanup step to recover back to 3 running machines and members if the test fails
```

Comment:
do we still have serial jobs where this could become an issue?

Reply:
Not currently no. But we may have other scaling related tests that may have to run serially in the future. But I'd prefer to leave that as a todo for when we actually have those.